repeater/src/test/resources/payload_fixed.json

```diff
@@ -0,0 +1,124 @@
+{
+  "eventId": "90aebb1b-1d49-455b-9049-15ec72dfe5a9",
+  "etlTstamp": "2020-11-10T16:56:39.283Z",
```

Comment:
I might be missing something, but I don't think there are `eventId` and `etlTstamp` anywhere in repeater's output.

Reply:
So, I'm also not sure where data like this should go, it's neither a bad row (not SDJ, nor failed insert that can be forwarded to BQ)